bedrock/firefox/templates/firefox/channel/desktop.html

```diff
@@ -2,17 +2,31 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% from 'macros.html' import google_custom_search with context %}
```

Comment:
The `extends` call should be first in the template. While this works it's best to move the `from` call here to below the `extends`.